bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Challenge/CreateChallenge.cs

```diff
@@ -0,0 +1,236 @@
+
+
+
+using ElectionGuard.Decryption.Accumulation;
+using ElectionGuard.Decryption.Extensions;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.Challenge;
+// TODO: async versions
+public static class CreateChallengeExtensions
```

Comment:
similar to the accumulate shares functions, and all of the other sections in this project, this class jsut holds the logic to loop over the objects and operate on them.